src/api/app/models/event/added_user_to_group.rb

```diff
@@ -0,0 +1,17 @@
+module Event
+  class AddedUserToGroup < Base
+    self.message_bus_routing_key = 'groups_user.create'
+    self.description = 'Added user to group'
+    payload_keys :group, :user, :who
+
+    receiver_roles :user
+
+    def subject
+      "You've been added to the group #{payload['group']}"
```

Comment:
```suggestion
      "The user #{payload['user']} has been added to the group #{payload['group']} by #{payload['who']}"
```

Reply:
I had rephrased the subject before I read your comment. It's finally: 

` "'#{payload['who']}' removed you from the group '#{payload['group']}'"`